plugins/woocommerce/includes/class-wc-brands.php

```diff
@@ -396,9 +400,7 @@ public function brand_description() {
 	public function show_brand() {
 		global $post;
 
-		$has_block_template = apply_filters( 'woocommerce_has_block_template', false, 'taxonomy-product_brand' ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
-
-		if ( is_singular( 'product' ) && ! $has_block_template ) {
+		if ( is_singular( 'product' ) && ! wc_current_theme_is_fse_theme() ) {
```

Comment:
Heads-up that a theme might be a block theme (so `wc_current_theme_is_fse_theme()` returns `true`) but we might be rendering the _Classic Template_ block, which in the end is the same as the PHP template.

So I think it would be better t o remove the check from here:

```suggestion
		if ( is_singular( 'product' ) ) {
```

And instead add:

```PHP
remove_action( 'woocommerce_product_meta_end', array( $this, 'show_brand' ) );
```

When we add the hooked block.

Reply:
Nice catch. Implemented in edf3e136a648a89995de79e6671aa5d96245fb6a